### PR TITLE
Moved run.stop() to before 'Explore experiment'

### PR DESCRIPTION
**Pull request by SiddhantSadangi**

### CODE REVIEW

**Comment by jakubczakon** [comment]
@SiddhantSadangi I see you removed the "Step 1: ..." from the examples. 
This is style-guide decision and I'd like to discuss it with @kamil-kaczmarek before we do anything. 
Could you revert that to "Step 1: ..."?

**Comment by jakubczakon** [comment]
@SiddhantSadangi could you re-run tests (Actions) and see if it fails or not. 
I'd like to close this but with many red-tests I don't think I can...

**Comment by SiddhantSadangi** [comment]
@jakubczakon , did you retrigger the action? I see one running currently--> [https://github.com/neptune-ai/examples/actions/runs/1142835565](https://github.com/neptune-ai/examples/actions/runs/1142835565)

**Comment by jakubczakon** [comment]
Yeah, I actually did that @SiddhantSadangi :)

**Comment by kamil-kaczmarek** [comment]
@SiddhantSadangi you do not have my fix here. PLs looks at requirements on master: https://github.com/neptune-ai/examples/tree/main/integrations-and-supported-tools/sklearn/scripts

for both scripts and notebooks it should be `0.24.1`.

**Comment by kamil-kaczmarek** [comment]
@SiddhantSadangi -> we got it :tada: tests passed

**Comment by jakubczakon** [comment]
@kamil-kaczmarek can we merge this?

**Comment by kamil-kaczmarek** [comment]
@jakubczakon I did not do any review of this PR. I only fixed sklearn connected error.

Can make the review if you need

File: how-to-guides/hello-world/notebooks/Neptune_hello_world.ipynb [hunk 1]
```diff
@@ -170,7 +170,7 @@
     "For example:\n",
     "\n",
     "```python\n",
-    "neptune.init(project='my_workspace/my_project', \n",
+    "neptune.init(project_qualified_name='my_workspace/my_project', \n",
```
Comment (kamil-kaczmarek) [inline how-to-guides/hello-world/notebooks/Neptune_hello_world.ipynb:173]:
```suggestion
    "neptune.init(project='my_workspace/my_project', \n",
```

File: how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb [hunk 2]
```diff
@@ -158,7 +158,7 @@
     "For example:\n",
     "\n",
     "```python\n",
-    "neptune.init(project='my_workspace/my_project', \n",
+    "neptune.init(project_qualified_name='my_workspace/my_project', \n",
```
Comment (kamil-kaczmarek) [inline how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb:161]:
```suggestion
    "neptune.init(project='my_workspace/my_project', \n",
```

**Comment by kamil-kaczmarek** [comment]
Just skimmed through it and it seems that there in `project_qualified_name` argument in the `neptune.init()` used muptiple times.

We now have `project` instead.

**Comment by kamil-kaczmarek** [comment]
changed merged in #70

**Pull request closed without merging**